### ISSUE TITLE
Show big image when clicked

### ISSUE THREAD

**Issue by niccokunzmann** [niccokunzmann/mundraub-android#43]
When I open the plant details and click on the image of the plant,
I would like to see this image in big.

**Comment by JakubPalsa** [comment]
Can I start working on this?

**Comment by niccokunzmann** [comment]
@JakubPalsa Yes, you can. You do not even need to ask. I described the process [here](https://github.com/niccokunzmann/mundraub-android/commit/602c29079e802f6ad446899f029f33d6b6b15fa5#diff-04c6e90faac2675aa89e2176d2eec7d8R35).
